apps/content/api/internal/assets_detail.py

```
@@ -37,13 +37,14 @@ class DetailAssetOut(Schema):
     thumbnail_url: AbsoluteUrl | None
     publisher: DetailAssetPublisherOut = Field(alias="publisher")
     license: LicenseChoice
+    is_open_access: bool
     snapshots: list[DetailAssetSnapshotOut] = Field(default_factory=list, alias="previews")
 
 
 @router.get("assets/{id}/", response=DetailAssetOut, auth=None)
 def detail_assets(request: Request, id: int):
```

Ruff (0.15.18)

[error] Function argument `id` is shadowing a Python builtin

(A002)